README.md

```
@@ -48,49 +48,66 @@
 | Thanks To | URL | Status |
 | --------- | --- | ------ |
+| catsmile  | https://files-axone.catsmile.space/snapshot_latest.tar.lz4 | LIVE     |
 
 > State / Quick Sync
 
 | Thanks To | URL | Status |
 | --------- | --- | ------ |
+| catsmile  | https://rpc-axone.catsmile.space | LIVE     |
+# State Sync Config (Axone via catsmile)
+
+STATE_SYNC_RPC=https://rpc-axone.catsmile.space
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)